contacts24/models/notes.py

```diff
@@ -1,24 +1,41 @@
-from collections import UserDict
-
-from contacts24.models.note import Note
-
-# Temporary contract
-class Notes(UserDict):
-    def add_note(self, note: Note) -> None:
-        # To complete by Sergii Shulga
-        pass
 
-def get_notes():
-    pass
 
 
-def create_note():
-    pass
-
+from collections import UserDict
+from contacts24.errors import NonExistingNote
 
-def update_note():
-    pass
 
 
-def delete_note():
-    pass
+class Notes(UserDict):
+    """This class provides functionality for working with notes"""
+    def add_note(self, title, text):
+        """Adds a new note to the notebook."""
+        max_index = max(self.data.keys()) if self.data else 0
+        note_id = max_index + 1  # Generates a unique identifier for the note
+        self.data[note_id] = {'note_id': note_id, 'title': title, 'text': text}
```

Comment:
@ShulgaSV 
Everything looks good

Consider using new class `Note` with `Title(Field)` `Text(Field)` and `Id(Field)`  instead of dictionary

Reply:
@ShulgaSV @SavchukR 
Let's leave this AS-IS, not to stay too long on this PR and start a new ticket.

Reply:
Yes
@ShulgaSV please proceed with @IhorNi scope

Reply:
@ShulgaSV this will be for the next PR
Currently closing this topic for now